plugins/file-open-save/core/src/main/javascript/app/components/files/files.component.js

```diff
@@ -183,14 +184,12 @@ define([
       vm.sortState = vm.sortField === field ? vm.sortState : 0;
       switch (vm.sortState) {
```

Comment:
@tgf This does not allow the user to clear all sorting. Meaning, 1st click will sort ascending. 2nd click will sort descending. 3rd click will sort ascending again and so forth. Does UX want the 3rd click to clear all sorting? If so, this will need to be fixed. If not, then we are good.